GitUI/CommandsDialogs/SettingsDialog/MergeToolsHelper.cs

```diff
@@ -402,8 +359,28 @@ private static string GetRegistryValue(RegistryKey root, string subkey, string k
             return value ?? string.Empty;
         }
 
-        private static string GetVisualStudioPath()
+        private static string GetVsDiffMergePath()
         {
+            //%ProgramFiles(x86)%\Microsoft Visual Studio\Installer\vswhere.exe
+            var vsVersionNumber = new string[] { "2020", "2019", "2018", "2017" };
+            var vsVersionType = new string[] { "Professional", "Community" };
```

Comment:
We all have bad days, I presume you had yours... Calling names won't get us anywhere.

Back to your PR - I can't approve, let alone merge, your PR in the current state. The VS check you've proposed is wrong.